### PR TITLE
Feature/notification

### PR DESCRIPTION
**Pull request by verygreenboi**

### CODE REVIEW

File: src/notification/send-notification.spec.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            to: "1234",
+            from: "1234",
+            message: "Hello",
+          },
+        };
+        await sendNotifications(clientName, port, [notification, notification]);
+      });
+
+      it("should call axios once", () => {
+        expect(spy).toBeCalledTimes(2);
```
Comment (horlabyc) [inline src/notification/send-notification.spec.ts:33]: Is this meant to call axios once or 2 times

File: src/notification/send-notification.spec.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            from: "1234",
+            message: "Hello",
+          },
+        };
+        await sendNotifications(clientName, port, notification);
+      });
+
+      it("should call spy with notification", () => {
+        expect(spy).toBeCalledWith(
+          `http://localhost:${port}/v1.0/publish/${clientName}/${UserNotificationTopic.USER_ONBOARDED}`,
```
Comment (horlabyc) [inline src/notification/send-notification.spec.ts:54]: Is this localhost url okay?

File: src/notification/send-notification.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  const promises: Promise<AxiosResponse>[] = [];
+  if (Array.isArray(notifications)) {
+    payloads = notifications;
+  } else {
+    payloads = [notifications];
+  }
+  payloads.forEach((payload) => {
+    promises.push(
+      axios.post(
+        `http://localhost:${clientPort}/v1.0/publish/${clientName}/${payload.topic}`,
```
Comment (horlabyc) [inline src/notification/send-notification.ts:19]: Is this localhost url okay?

File: src/notification/send-notification.spec.ts (hunk 1 above)
Comment (verygreenboi) [reply src/notification/send-notification.spec.ts:33]: The function takes a notification or an array of notifications

File: src/notification/send-notification.spec.ts (hunk 2 above)
Comment (verygreenboi) [reply src/notification/send-notification.spec.ts:54]: Yes. Dapr loads a sidecar

File: src/notification/send-notification.ts (hunk 3 above)
Comment (verygreenboi) [reply src/notification/send-notification.ts:19]: Yes. Same answer

**Review by horlabyc — APPROVED** [review]

**Pull request merged**